clippy.toml

```diff
@@ -2,4 +2,5 @@ disallowed-methods = [
   "xtra::Address::do_send_async", # discards the return value, possibly swallowing an error
   "xtra::Address::do_send", # discards the return value, possibly swallowing an error
   "xtra::message_channel::MessageChannel::do_send", # discards the return value, possibly swallowing an error
+  "xtra::Context::notify_interval", # does not wait for response from the previous handler, piling up messages
```

Comment:
It would be good to use the syntax here that allows us to specify the reason so it helps the developer better.

Perhaps also point to the function to use instead?

Reply:
I'll create a separate PR to change to the new clippy syntax for all of them. Whichever comes later, I'll rebase the other one to align with your intention.

(added mention of the preferred method)